.github/workflows/test-build.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       matrix:
         # We want to make sure dskit can support multiple golang versions
         # by ensuring the test would pass using all these supported versions.
-        go-version: ['1.21.x', '1.22.x']
+        go-version: ['1.21.x', '1.22.x', '1.23.x']
```

Comment:
Is this required? If not, let's do that in separate PR.

Reply:
Somewhat required, Loki already uses Go 1.23. A separate PR would have been better, but I don't want to wait another 25 minutes for the CI to run :sweat_smile: